pubsub/snapshot.go

```diff
@@ -42,11 +43,32 @@ func (s *Snapshot) ID() string {
 	return s.name[slash+1:]
 }
 
+// SetLabels replaces the current set of labels completely with the new set.
```

Comment:
Maybe "SetLabels set or replaces the labels on a given snapshot."

The existing docstring seems overly focused on the replace case.